cluster/gke/util.sh

```diff
@@ -284,13 +289,19 @@ function detect-node-names {
 #   ZONE
 #   CLUSTER_NAME
 # Vars set:
-#   NODE_INSTANCE_GROUP
+#   NODE_INSTANCE_GROUPS
 function detect-node-instance-group {
   echo "... in gke:detect-node-instance-group()" >&2
-  local url=$("${GCLOUD}" ${CMD_GROUP:-} container clusters describe \
+  local urls+=$("${GCLOUD}" ${CMD_GROUP:-} container clusters describe \
     --project="${PROJECT}" --zone="${ZONE}" \
     --format='value(instanceGroupUrls)' "${CLUSTER_NAME}")
-  NODE_INSTANCE_GROUP="${url##*/}"
+  urls=(${urls//;/ })
+  NODE_INSTANCE_GROUPS=()
+  if [[ -n "${urls[@]:-}" ]]; then
```

Comment:
Will the for loop not just exit immediately if you do `for url in "${urls[@]:-}"; do` at which point you could remove this if check?